nearswap/tests/nep21_transfer.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(unused)]
```

Comment:
Why not moving this this to `other-contracts/nep-21/test`?

Reply:
I see, we move it from the simulation_test.rs. Let's move it to the location suggested above.

Reply:
OK